### PR TITLE
docs: add lambdos endpoints

### PR DESCRIPTION
**Pull request by manumena**

### CODE REVIEW

File: openapi/components/schemas/lambdas/200-lands-paginated.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        y:
+          type: string
+        image:
+          type: string
+        description:
+          type: string
+  pageNum:
+    type: number
+  pageSize:
+    type: number
```
Comment (aleortega) [inline openapi/components/schemas/lambdas/200-lands-paginated.yaml:29]: Both properties are set as number but the example returns string

**Review by aleortega — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/lambdas/200-names-paginated.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        contractAddress:
+          type: string
+        tokenId:
+          type: string
+        price:
+          type: string
+  pageNum:
+    type: number
+  pageSize:
+    type: number
```
Comment (aleortega) [inline openapi/components/schemas/lambdas/200-names-paginated.yaml:19]: Same here

File: openapi/components/schemas/lambdas/200-wearables-paginated.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        urn:
+          type: string
+        amount:
+          type: number
+  totalAmount:
+    type: number
+  pageNum:
+    type: number
+  pageSize:
+    type: number
```
Comment (aleortega) [inline openapi/components/schemas/lambdas/200-wearables-paginated.yaml:17]: 👼

File: openapi/components/examples/lambdas/200-wearables-paginated.json [hunk 4]
```diff
@@ -0,0 +1,93 @@
+{
+    "value": {
+        "wearables": [
+            {
+                "urn": "urn:decentraland:matic:collections-v2:0x6d83706836e820e79252edd48da419d2d03592d4:0",
+                "individualData": [
```
Comment (aleortega) [inline openapi/components/examples/lambdas/200-wearables-paginated.json:6]: Are we missing this property in the schema?

Comment (manumena) [reply openapi/components/examples/lambdas/200-wearables-paginated.json:6]: yes thanks!

**Review by aleortega — APPROVED** [review]

**Pull request merged**